Healthy/Classes/ReusableViews/MainTabBar/MainTabBar.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+
+class MainTabBar: UITabBar {
+    
+    private enum Constants {
```

Comment:
Make the name more descriptive -> MainTabBarConstants

Reply:
I think the current name matches the convention. Only one step is to add it in a private extension after the end of the file.